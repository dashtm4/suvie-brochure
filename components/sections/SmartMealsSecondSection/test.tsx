import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { SmartMealsSecondSection } from '.'

describe('<SmartMealsSecondSection />', () => {
  it('should renders correct <SmartMealsSecondSection />', () => {
    const wrapper = shallow(<SmartMealsSecondSection />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
