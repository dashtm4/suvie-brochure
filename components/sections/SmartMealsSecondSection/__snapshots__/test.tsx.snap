// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SmartMealsSecondSection /> should renders correct <SmartMealsSecondSection /> 1`] = `
<section
  className="relative pt-6 md:pb-20 md:pt-22"
>
  <Layout>
    <div
      className="mb-28 text-center"
      role="presentation"
    >
      <Image
        alt=""
        height={75}
        src="/images/smart-meals-page/smart-meals--icon.svg"
        width={75}
      />
      <h2
        className="mb-5 font-serif text-3xl font-normal text-secondary"
      >
        Suvie
        <span
          className="font-medium"
        >
           
          Smart Meals
        </span>
      </h2>
      <div
        className="mx-auto max-w-xl font-serif text-2xl font-normal md:text-4xl text-secondary"
      >
        Save even more time when you combine the Suvie appliance with our chef-designed meals.
      </div>
    </div>
    <VideoTabs
      activeIndex={0}
      direction="reverse"
      itemPosters={
        Array [
          "",
          "",
          "",
        ]
      }
      itemVideos={
        Array [
          "https://embed-ssl.wistia.com/deliveries/43aa58e07c0d7fd6995d412c740d1d1e5b675654.m3u8",
          "https://embed-ssl.wistia.com/deliveries/338193c2fca6de3fde33a569aac909233aa285d1.m3u8",
          "https://embed-ssl.wistia.com/deliveries/e35c0901e38c7532b2abf79b73d5d1bfb9e9f76f.m3u8",
        ]
      }
      setActiveIndex={[Function]}
      tabData={
        Object {
          "items": Array [
            Object {
              "description": "Choose between Flex, Weekly, or Protein based on your individual preferences. Pause, cancel or change your plan at any time. ",
              "imageSrc": "",
              "title": "Choose Plan",
              "videoSrc": "https://embed-ssl.wistia.com/deliveries/43aa58e07c0d7fd6995d412c740d1d1e5b675654.m3u8",
            },
            Object {
              "description": "Use our meal planner to mix and match your perfect combination of meals from our 60+ options. Filter by your health and taste preferences. ",
              "imageSrc": "",
              "title": "Order Meals",
              "videoSrc": "https://embed-ssl.wistia.com/deliveries/338193c2fca6de3fde33a569aac909233aa285d1.m3u8",
            },
            Object {
              "description": "Smart Meals come in special packaging that tells Suvie how to cook each ingredient. Just scan, load, and let Suvie handle the rest.",
              "imageSrc": "",
              "title": "Scan & Cook",
              "videoSrc": "https://embed-ssl.wistia.com/deliveries/e35c0901e38c7532b2abf79b73d5d1bfb9e9f76f.m3u8",
            },
          ],
          "title": "Gourmet & Ready to Cook",
        }
      }
    />
  </Layout>
</section>
`;
